front/src/reducers/osrdconf/operationalStudiesConf/utils.ts

```diff
@@ -83,7 +83,7 @@ export const addPathStep = (pathSteps: PathStep[], newPathStep: PathStep): PathS
     const index = pathSteps.findIndex(
       (step) => step.positionOnPath && step.positionOnPath > newPathStep.positionOnPath!
     );
-    return addElementAtIndex(pathSteps, index, newPathStep);
+    return addElementAtIndex(pathSteps, index, { ...newPathStep, isFromPowerRestriction: true });
```

Comment:
I'm a bit antsy to set this flag in a generic `addPathStep()` function… Could we maybe set it from the caller side?

Reply:
Done